seeders/06_cabs.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+    up: queryInterface => {
```

Comment:
We could use faker to seed data

Reply:
Haven't used faker yet, will give it a go!